Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -78,7 +79,7 @@ extension Card {
         return NSRegularExpression(staticPattern: "^3[47]\\d{13}$")
       case .discover:
         // swiftlint:disable:next line_length
-        return NSRegularExpression(staticPattern: "^6(011(0[0-9]|[2-4]\\d|74|7[7-9]|8[6-9]|9[0-9])|4[4-9]\\d{3}|5\\d{4})\\d{10}$")
+        return NSRegularExpression(staticPattern: "^65[4-9][0-9]{13}|64[4-9][0-9]{13}|6011[0-9]{12}|(622(?:12[6-9]|1[3-9][0-9]|[2-8][0-9][0-9]|9[01][0-9]|92[0-5])[0-9]{10})$")
```

Comment:
please add unit test for new changes. it's hard to know if the new reg exp is right for `.discover`.

Reply:
Added